models/components/tokenizers/utils.py

```diff
@@ -92,4 +92,4 @@ def render_token(t: bytes) -> str:
     """Pretty print a token, escaping control characters."""
     s = t.decode("utf-8", errors="replace")
     s = replace_control_characters(s)
-    return s
\ No newline at end of file
+    return s
```

Comment:
my linter did this :) I really wish everyone would actually use linters... otherwise i have to do a big linting run and it pollutes new prs

Reply:
Not saying that I always use it tho lol, it isn't working on my remote